src/storage/mod.rs

```diff
@@ -104,13 +104,28 @@ impl Mutation {
     }
 }
 
+/// Represents the status of a transaction.
+#[derive(PartialEq, Debug)]
+pub enum TxnStatus {
+    /// The txn is just rolled back by the current command.
+    Rollbacked,
+    /// The txn was already rolled back before.
+    RollbackedBefore,
+    /// The txn doesn't exist.
+    NotExist,
+    /// The txn haven't yet been committed. Carries the primary lock's TTL.
+    Uncommitted(u64),
```

Comment:
I prefer to use a struct like enum variant rather than the tuple like one here. Then, we don't need to check the comments here and never worry about misunderstanding the meaning of the value.